### ISSUE TITLE
Naming of class BridgeConstructionElement

### ISSUE THREAD

**Issue by TatjanaKutzner** [opengeospatial/CityGML-3.0CM#21]
The modules Building and Tunnel define the classes _BuildingConstructiveElement_ and _TunnelConstructiveElement_, both are subclasses of _AbstractConstructiveElement_. The classes allow for mapping constructive elements from BIM datasets onto CityGML and are new in CityGML 3.0. 

The module Bridge defines the class _BridgeConstructionElement_. This class already exists in CityGML 2.0 for modelling elements which are essential from a structural point of view. The class was kept in CityGML 3.0, but is now also modelled as subclass of AbstractConstructiveElement. 

For this reason, I suggest to change the naming of _BridgeConstructionElement_ to _BridgeConstructiveElement_ to be consistent with _BuildingConstructiveElement_ and _TunnelConstructiveElement_.

**Comment by TatjanaKutzner** [comment]
In today's webconference we opted for changing the name of the class to "BridgeConstructiveElement". 

However, the semantics of the class "BridgeConstructionElement" in CityGML 2.0 differs slightly, as it is used for elements that are essential from a structural point of view. 

To be able to express this characteristic also after renaming the class, we decided to add an attribute "isStructuralElement" of the type "Boolean" to the class "AbstractConstructiveElement". In this way, the characteristic is also inherited by "BuildingConstructiveElement" and "TunnelConstructiveElement".
